src/main/cliapp/src/components/Dashboard.js

```diff
@@ -52,6 +53,10 @@ export const Dashboard = () => {
           setMACount(results.totalResults);
         });
 
+        searchService.search('zfaTerm', 0, 0).then(results => {
```

Comment:
All the other endpoints this is lowercase can we make it `zfaterm` to be consistant with the rest of them?

Reply:
I'd rather change all the other endpoints to use camel case as it is easier to read.

Reply:
https://datatracker.ietf.org/doc/html/rfc3986

"Although schemes are case-
   insensitive, the canonical form is lowercase and documents that
   specify schemes must do so with lowercase letters.  An implementation
   should accept uppercase letters as equivalent to lowercase in scheme
   names (e.g., allow "HTTP" as well as "http") for the sake of
   robustness but should only produce lowercase scheme names for
   consistency."

Reply:
I would be fine with: `term/mp`, `mp_term`, or `mp-term`... just not uppercase in URL's.

Reply:
Ok, I'll change them all into a schema:
term/zfa
term/ma
... 
Will do that in a subsequent PR. Please let this one then go through for testing purposes.

Reply:
So can we go the other direction, as in, lowercase this. So that its consistent with ALL the other ontology endpoints. And add the other PR later.